excretor/src/dbmodels.rs

```diff
@@ -25,3 +25,15 @@ pub struct DBChannel {
     pub topic: Option<String>,
     pub purpose: Option<String>,
 }
+
+#[derive(Serialize, Deserialize)]
+pub struct DBUser {
```

Comment:
This struct is the same as the `User` struct in `models.rs`. You can use that one.

Reply:
There is an Optional attribute though `pub image_url: Option<String>`

Reply:
Hmm, interesting. Ig the default value used to work before the `query!` macro was added. Fine then. But do change the from function to use the `From<T>` trait.